File: module06/module025/module2/src/main/java/com/android/example/module06_module025_module2/ClassAAA.java
package com.android.example.module06_module025_module2;

public class ClassAAA {
	private 	io.reactivex.subscribers.TestSubscriber<Object> instance_var_1_0 = new io.reactivex.subscribers.TestSubscriber();
	private 	io.reactivex.subscribers.TestSubscriber<Object> instance_var_1_1 = new io.reactivex.subscribers.TestSubscriber();
	private 	io.reactivex.subscribers.TestSubscriber<Object> instance_var_1_2 = new io.reactivex.subscribers.TestSubscriber();
	private 	io.reactivex.subscribers.TestSubscriber<Object> instance_var_1_3 = new io.reactivex.subscribers.TestSubscriber();

	public void method0(
		io.reactivex.subscribers.TestSubscriber<Object> param0,
		io.reactivex.subscribers.TestSubscriber<Object> param1) throws Throwable {
		param0.dispose();

		param0.requestMore(455567973l);

		for (int iAb = 0; iAb < 6; iAb++) {
			io.reactivex.subscribers.TestSubscriber<Object> local_var_3_0 = new io.reactivex.subscribers.TestSubscriber();
			local_var_3_0.isCancelled();

			local_var_3_0.assertNotSubscribed();

		}
		param0.hasSubscription();

	}

	public void method1(
		io.reactivex.subscribers.TestSubscriber<Object> param0,
		io.reactivex.observers.TestObserver<Object> param1) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_2 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_2.assertSubscribed();

		io.reactivex.internal.disposables.ListCompositeDisposable local_var_2_3 = new io.reactivex.internal.disposables.ListCompositeDisposable();
		local_var_2_3.isDisposed();

	}

	public void method2(
		io.reactivex.subscribers.TestSubscriber<Object> param0,
		io.reactivex.subscribers.TestSubscriber<Object> param1,
		io.reactivex.subscribers.TestSubscriber<Object> param2,
		io.reactivex.subscribers.TestSubscriber<Object> param3) throws Throwable {
		io.reactivex.internal.disposables.ListCompositeDisposable local_var_2_4 = new io.reactivex.internal.disposables.ListCompositeDisposable();
		local_var_2_4.isDisposed();

		param0.assertNotSubscribed();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_5 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_5.assertNotSubscribed();

	}

	public void method3(
		io.reactivex.subscribers.TestSubscriber<Object> param0,
		io.reactivex.internal.disposables.ListCompositeDisposable param1,
		io.reactivex.subscribers.TestSubscriber<Object> param2) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.requestMore(-523851691l);

		param0.assertSubscribed();

	}
}

File: module07/module21/module01/module3/src/main/java/com/android/example/module07_module21_module01_module3/ClassAAA.kt
package com.android.example.module07_module21_module01_module3

@Suppress("UNUSED_PARAMETER")
class ClassAAA {
	private 	val instance_var_1_0: com.android.example.module07_module21_module01_module2.ClassAAF = com.android.example.module07_module21_module01_module2.ClassAAF()
	private 	val instance_var_1_1: com.android.example.module07_module21_module01_module2.ClassAAI = com.android.example.module07_module21_module01_module2.ClassAAI()
	private 	val instance_var_1_2: com.android.example.module07_module21_module01_module5.ClassAAE = com.android.example.module07_module21_module01_module5.ClassAAE()

	fun method0(
		param0: com.android.example.module07_module21_module01_module5.ClassAAA,
		param1: com.android.example.module07_module21_module01_module2.ClassAAE
	) {
		for (iAb in 0..3) {
			val local_var_3_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
			local_var_3_0.cancel()

			val local_var_3_1: io.reactivex.disposables.SerialDisposable = io.reactivex.disposables.SerialDisposable()
			local_var_3_1.isDisposed()

			local_var_3_0.isCancelled()

		}
		val local_var_2_2: io.reactivex.observers.TestObserver<kotlin.Any> = io.reactivex.observers.TestObserver()
		local_var_2_2.hasSubscription()

		val local_var_2_3: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
		local_var_2_3.clear()

		val local_var_2_4: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_4.assertNotSubscribed()

	}

	fun method1(
		param0: com.android.example.module07_module21_module01_module5.ClassAAC,
		param1: com.android.example.module07_module21_module01_module5.ClassAAB,
		param2: com.android.example.module07_module21_module01_module5.ClassAAB,
		param3: com.android.example.module07_module21_module01_module5.ClassAAI
	) {
		val local_var_2_4: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_4.onError(kotlin.Throwable())

		val local_var_2_5: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_5.cancel()

	}

	fun method2(
		param0: com.android.example.module07_module21_module01_module2.ClassAAI,
		param1: com.android.example.module07_module21_module01_module5.ClassAAF,
		param2: com.android.example.module07_module21_module01_module2.ClassAAG,
		param3: com.android.example.module07_module21_module01_module2.ClassAAA
	) {
		val local_var_2_4: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_4.requestMore(2464324119584869059)

		val local_var_2_5: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_5.onComplete()

		val local_var_2_6: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_6.dispose()

	}

	fun method3(
		param0: com.android.example.module07_module21_module01_module2.ClassAAF,
		param1: com.android.example.module07_module21_module01_module2.ClassAAG,
		param2: com.android.example.module07_module21_module01_module2.ClassAAI
	) {
		val local_var_2_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_3.request(-6852247515783905960)

	}

	fun method4(
		param0: com.android.example.module07_module21_module01_module5.ClassAAJ,
		param1: com.android.example.module07_module21_module01_module5.ClassAAE,
		param2: com.android.example.module07_module21_module01_module5.ClassAAG,
		param3: com.android.example.module07_module21_module01_module5.ClassAAG
	) {
		val local_var_2_4: io.reactivex.internal.util.OpenHashSet<kotlin.Any> = io.reactivex.internal.util.OpenHashSet()
		local_var_2_4.keys()

		val local_var_2_5: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_5.requestMore(2387052764061353003)

	}
}
